.github/workflows/main.yml

```diff
@@ -60,11 +63,32 @@ jobs:
         fetch-depth: 0
     - name: Configure AWS Credentials
       continue-on-error: true
+      if: github.repository == 'ray-project/ray'
       uses: aws-actions/configure-aws-credentials@v1
       with:
         aws-access-key-id: ${{ secrets.GHA_AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ secrets.GHA_AWS_SECRET_ACCESS_KEY }}
         aws-region: us-west-2
+    - name: Prepare caching
+      id: info
+      shell: bash
+      run: |
+        case "${OSTYPE}" in linux*) pip_cache=~/.cache/pip;; darwin*) pip_cache=~/Library/Caches/pip;; msys) pip_cache="${LOCALAPPDATA}\pip\Cache";; esac
```

Comment:
can you make this multiline for more readability?

Reply:
Sure